js/pricing.js

```diff
@@ -1,3 +1,6 @@
+---
+---
+// Do not remove `---` tags. These are the frontmatter tags for Jekyll variables.
```

Comment:
Can we embed this comment into the tags themselves?

Reply:
Unfortunately it will break a Jekyll build.